tests/fixtures/commons.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
-const assign = require('lodash.assign');
+const merge = require('lodash.merge');
 
 const output = { name: '' };
 
-assign(output, { name: 'common' });
+merge(output, { name: 'common' });
```

Comment:
I believe this should not be changed, since it is part of the tests. The fixes you had to make in the tests are caused by this, not by changing `assign` to `merge` in the `_writeOutput` method.

I actually find it weird that this change is affecting the current tests, but I'd like for this PR to focus on one matter at a time. If you could please revert this specific change (and the tests), or provide a reason for that change, it would be great.